go-controller/pkg/util/net.go

```diff
@@ -317,11 +318,11 @@ func IPsToNetworkIPs(ips ...*net.IPNet) []*net.IPNet {
 	return nets
 }
 
-// StringsToIPs takes a slice of strings and returns a slice of net.IPs
-func StringsToIPs(ips []string) []net.IP {
-	s := make([]net.IP, len(ips))
-	for i := range ips {
-		s[i] = net.ParseIP(ips[i])
+// IPsToStringSet takes a slice of IPs and returns a string set
+func IPsToStringSet(ips []net.IP) sets.Set[string] {
```

Comment:
This could be a generic method, very similar to the already existing one `StringSlice`

Reply:
oh good point, I always forget we have that, I think I should be able to just use that lmc since we do have a .String() for ip net.

Reply:
done.